assets/.ls-lint.yml

```diff
@@ -25,3 +21,4 @@ ignore:
   - dist
   - .openapi
   - src/@types
+  - src/modules/database/migrations
```

Comment:
All migration files are using different convention:

Eg.
`20230302162022_create_table_users.ts`

We can either rename migrations file name, support this format or ignore their folder completely.

Reply:
```suggestion
   src/modules/database/migrations:
    .ts: regex:([.a-z0-9]*)([_.][a-z0-9]+)*
```